src/subscript/fmu_copy_revision/fmu_copy_revision.py

```diff
@@ -35,6 +35,10 @@
 
     fmu_copy_revision --source 21.0.0  (...other options are defaulted)
 
+Backup:
```

Comment:
Should we include this? It is maybe better to get users to report errors that can be fixed?

Reply:
Yes the intention was just if the user got panic. I can reformulate a bit